src/main/java/com/openshift/internal/restclient/DefaultClient.java

```diff
@@ -109,18 +109,29 @@ public IWatcher watch(String namespace, IOpenShiftWatchListener listener, String
 	}
 
 	@Override
+	public IWatcher watch(IOpenShiftWatchListener listener, String...kinds) {
+		return this.watch("", listener, kinds);
```

Comment:
it can be, and is unlikely to work for normal users (just like the list below, I guess)